controllers/storagecluster/storageclasses.go

```diff
@@ -249,11 +249,10 @@ func newCephBlockPoolStorageClassConfiguration(initData *ocsv1.StorageCluster) S
 	}
 }
 
-// newCephNFSStorageClassConfiguration generates configuration options for a Ceph Filesystem StorageClass.
+// newCephNFSStorageClassConfiguration generates configuration options for a Ceph NetworkFilesystem StorageClass.
 func newCephNFSStorageClassConfiguration(initData *ocsv1.StorageCluster) StorageClassConfiguration {
 	persistentVolumeReclaimDelete := corev1.PersistentVolumeReclaimDelete
-	// VolumeExpansion is not yet supported for nfs.
-	allowVolumeExpansion := false
+	allowVolumeExpansion := true
```

Comment:
Have you checked upgrade ? 

is this field mutable or whether ocs-op deletes and recreates sc  ?

Reply:
Have to check on this

Reply:
checked, its mutable